pulseapi/entries/views.py

```diff
@@ -168,6 +168,8 @@ class EntriesListView(ListCreateAPIView):
     search_fields = (
         'title',
         'description',
+        'tags__name',
+        'creators__name',
```

Comment:
we probably need to update https://github.com/mozilla/network-pulse-api/blob/master/pulseapi/entries/views.py#L152 so that there's clear documentation on how to actually do searching for these four properties?